app/jobs/submit_scan_request_job.rb

```diff
@@ -11,6 +11,8 @@ def perform(scan)
 
     scan.update(illiad_response_data: JSON.parse(response.body).select { |_, value| value.present? } || {})
 
+    scan.notify_ilb! if scan.illiad_response_data['Message']
```

Comment:
I'm wondering if we should add some sort of accessor into either `IlliadRequest` or the `Scan` class to let us know that the response was an error (so the concept of an Illiad error doesn't only exist in this job)

Reply:
Sounds good. I moved it to the Scan class